shacl12-test-suite/tests/core/node/manifest.ttl

```diff
@@ -23,6 +23,7 @@
 	mf:include <maxExclusive-001.ttl> ;
 	mf:include <maxInclusive-001.ttl> ;
 	mf:include <maxLength-001.ttl> ;
+	mf:include <memberShape-001.ttl> ;
```

Comment:
I see the new files `memberShape-00{1,2,3,4}.ttl`; but, only `...001.ttl` is listed.  Missed?

Reply:
Whoops, please disregard, I mixed up the folder.